examples/BodyPose-blazepose-skeleton/sketch.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2024 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+let video;
+let bodyPose;
+let poses = [];
+let connections;
+
+function preload() {
+  // Load the bodyPose model
+  bodyPose = ml5.bodyPose("BlazePose");
+}
+
+function setup() {
+  createCanvas(640, 480);
+
+  // Create the video and hide it
+  video = createCapture(VIDEO);
+  video.size(width, height);
+  video.hide();
+
+  // Start detecting poses in the webcam video
+  bodyPose.detectStart(video, gotPoses);
+  //get the skeleton connection information
+  connections = bodyPose.getSkeleton();
+}
+
+function draw() {
+  // Draw the webcam video
+  image(video, 0, 0, width, height);
+
+  //draw the skeleton connections
+  for (let i = 0; i < poses.length; i++) {
+    let pose = poses[i];
+    for (let j = 0; j < connections.length; j++) {
+      let pointAIndex = connections[j][0];
```

Comment:
I wonder if there is a naming convention that might be more clear than using a 2D array (which can be very confusing for beginners as well as not being descriptive). would this be better:

```js
let pointAIndex = connections[j].a
let pointAIndex = connections[j].b
```

Is there something more descriptive? Also, we'll need to add this to the documentation

Reply:
It might help to add an intermediate variable?
```
let currentConnection = connections[j];
let pointAIndex = currentConnection[0];
let pointBIndex = currentConnection[1];
```

(If this was not for beginners I would definitely use destructuring `let [pointAIndex, pointBIndex] = connections[j]`)